test/tests/workspace/k3s_test.go

```diff
@@ -72,13 +79,14 @@ func TestK3s(t *testing.T) {
 				"KUBECONFIG=/etc/rancher/k3s/k3s.yaml",
 			}
 			var respWaitForK3s agent.ExecResponse
+			timeout := fmt.Sprintf("%.0fm", TIME_OUT.Minutes())
 			err = rsa.Call("WorkspaceAgent.Exec", &agent.ExecRequest{
 				Dir:     "/",
 				Command: "bash",
 				Env:     kubeEnv,
 				Args: []string{
 					"-c",
-					"timeout 30s bash -c 'while [ ! -e /etc/rancher/k3s/k3s.yaml ]; do sleep 1; done' && sudo chmod +r /etc/rancher/k3s/k3s.yaml && timeout 1m bash -c 'until /workspace/k3s kubectl wait --for=condition=Ready nodes -l node-role.kubernetes.io/master=true --timeout 30s; do sleep 1; done'",
+					fmt.Sprintf("timeout %s bash -c 'while [ ! -e /etc/rancher/k3s/k3s.yaml ]; do sleep 1; done' && sudo chmod +r /etc/rancher/k3s/k3s.yaml && timeout %s bash -c 'until /workspace/k3s kubectl wait --for=condition=Ready nodes -l node-role.kubernetes.io/master=true --timeout %s; do sleep 1; done'", timeout, timeout, timeout),
```

Comment:
The main change in this PR is to extend the timeout from 30secs/1mins to 5 mins, correct?

Reply:
@jenting 
Yes, there should be no need to be short